ports/unix/main.c

```diff
@@ -532,8 +532,8 @@ MP_NOINLINE int main_(int argc, char **argv) {
     {
         MP_DECLARE_CONST_FUN_OBJ_0(extra_coverage_obj);
         MP_DECLARE_CONST_FUN_OBJ_0(extra_cpp_coverage_obj);
-        mp_store_global(QSTR_FROM_STR_STATIC("extra_coverage"), MP_OBJ_FROM_PTR(&extra_coverage_obj));
-        mp_store_global(QSTR_FROM_STR_STATIC("extra_cpp_coverage"), MP_OBJ_FROM_PTR(&extra_cpp_coverage_obj));
```

Comment:
One could argue that `QSTR_FROM_STR_STATIC(...)` used here is a coverage test for that macro... because otherwise it's never used.  But maybe it's worth removing this macro altogether, since as you point out it should anyway be the compile-time value.

Reply:
Moreover it's actaully alomost the same as qstr_from_str (I doubt the compiler is going to generate different code).

Reply:
I think the idea was that for static strings the strlen is known at compile time so can call `qstr_from_strn(str, len)` directly with a constant for the length.

Reply:
Probably, but I'm not sure how many compilers implement that. In any case I think removing is better for consistency and less confusing to have 2 ways to do almost the same thing.